presto-parser/src/main/java/com/facebook/presto/sql/tree/FunctionCall.java

```diff
@@ -24,50 +24,58 @@
 {
     private final QualifiedName name;
     private final Optional<Window> window;
+    private final Optional<Expression> filterClause;
     private final boolean distinct;
     private final List<Expression> arguments;
 
     public FunctionCall(QualifiedName name, List<Expression> arguments)
     {
-        this(Optional.empty(), name, Optional.<Window>empty(), false, arguments);
+        this(Optional.empty(), name, Optional.<Window>empty(),  Optional.<Expression>empty(), false, arguments);
```

Comment:
These can all be `Optional.empty()`. The existing `Optional.<Window>empty()` could also be simplified. (IntelliJ warns you about this)